mobile/src/main/java/org/openhab/habdroid/util/Util.java

```diff
@@ -155,6 +161,17 @@ public static void setActivityTheme(@NonNull final Activity activity, String the
             themeRes = R.style.HABDroid_Light;
         }
         activity.setTheme(themeRes);
+
+        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP) {
+            Bitmap icon = BitmapFactory.decodeResource(activity.getResources(), R.mipmap.icon_round);
+            TypedValue typedValue = new TypedValue();
+            Resources.Theme themeActivity = activity.getTheme();
```

Comment:
The theme is set in L163, but as res int, not as Theme. Can I convert res id to theme or should I use the getter here?

Reply:
Using the getter is fine, I think.

Reply:
No, changed it.